### PR TITLE
zoombot: deauthorization

### PR DESCRIPTION
**Pull request by marceloneil**

### CODE REVIEW

File: zoombot/zoombot/db.go [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+		OAuthDB: base.NewOAuthDB(db),
+	}
+}
+
+func (d *DB) CreateUser(userID, accountID, identifier string) error {
+	return d.RunTxn(func(tx *sql.Tx) error {
+		_, err := tx.Exec(`
+			INSERT INTO user
+			(user_id, account_id, identifier)
+			VALUES (?, ?, ?)
```
Comment (joshblum) [inline zoombot/zoombot/db.go]: duplicate key?

File: zoombot/zoombot/api.go [hunk 2]
```diff
@@ -49,6 +51,36 @@ type GlobalDialInNumbers struct {
 	Type        string `json:"type"`
 }
 
+// Get User
+
+type GetUserResponse struct {
```
Comment (joshblum) [inline zoombot/zoombot/api.go]: why dont' we just have the fields we need so we never parse any user data we don't need

Comment (marceloneil) [reply zoombot/zoombot/api.go]: I set this up so we could easily use these functions for whatever, but that's a good point. I'll get rid of the unnecessary stuff. Do you think I should comment it out or just axe it entirely?

Comment (marceloneil) [reply zoombot/zoombot/api.go]: Never mind, I'm just going to axe the extra stuff and add a comment.

Comment (joshblum) [reply zoombot/zoombot/api.go]: perfect!

**Pull request merged**